rpc/dial_options.go

```diff
@@ -280,3 +283,10 @@ func WithForceDirectGRPC() DialOption {
 		o.disableDirect = false
 	})
 }
+
+// WithConn provides a preexisting connection to use. This option forces the webrtcSignalingAnswerer to not dial or manage a connection.
+func WithConn(conn ClientConn) DialOption {
```

Comment:
maybe ExternalSignalerConn? we should be clear that this would be used for the external signaler answerer